bandh/static/js/main.js

```diff
@@ -1804,33 +1836,34 @@
                         $.ajax({
                             url: "//app.tvpage.com/api/channels/"+channelId+"/videos",
                             dataType: 'jsonp',
+                            async:false,

```

Comment:
because it was clonning the video grid, and loading the same videos over and over